py/client/pydeephaven/_console_service.py

```diff
@@ -23,9 +23,12 @@ def start_console(self):
             if not self.console_id:
                 try:
                     result_id = self.session.make_ticket()
-                    response = self._grpc_console_stub.StartConsole(
-                        console_pb2.StartConsoleRequest(result_id=result_id, session_type=self.session._session_type),
+                    response, call = self._grpc_console_stub.StartConsole.with_call(
+                        console_pb2.StartConsoleRequest(
+                            result_id=result_id,
+                            session_type=self.session._session_type),
                         metadata=self.session.grpc_metadata)
+                    self.session.update_metadata(call.initial_metadata())
```

Comment:
Maybe a stupid question due to the lack of understanding of how the server's whole auth token logic works, but do we need to do this here? If a session is created with never_timeout = False, are we making it never expire? And if a session is created with never_timeout = True, should we count on the keep alive logic to update the auth token?

Reply:
What about a client app that only makes calls to the methods in the arrow flight service?

Reply:
That should work, we are also doing the metadata sending on those, look at the changes in `_arrow_flight_service.py`